gatsby-config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     {
       resolve: 'gatsby-source-strapi',
       options: {
-        apiURL: `${process.env.GATSBY_STRAPI_HOST}`,
```

Comment:
I'd like to keep the string literal version, just in case I mess up the Environment variables...